File: wls-eai-service/src/main/java/de/muenchen/oss/wahllokalsystem/eaiservice/domain/ergebnismeldung/AWerte.java
package de.muenchen.oss.wahllokalsystem.eaiservice.domain.ergebnismeldung;

import jakarta.persistence.Embeddable;
import lombok.Data;

@Data
@Embeddable
public class AWerte {

    private Long a1;
    
    private Long a2;

}

File: wls-gui-wahllokalsystem/src/types/ergebnisermittlung/AWerte.ts
import type { BezirkUndWahlID } from "@/types/ergebnismeldung/BezirkUndWahlID.ts";

export interface AWerte {
  bezirkUndWahlID: BezirkUndWahlID;
  a1: number;
  a2: number | null;
}
